### PR TITLE
feat(Q-dance): add cover

### PR DESCRIPTION
**Pull request by DarkVillager**
**Describe the changes in this pull request:**
- Added cover for q-dance & volume meter in precentage as small image key
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

</details>

### CODE REVIEW

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
After a discussion with my team, we have determined that having the volume displaying is unnecessary. Please remove this

File: websites/Q/Q-dance/presence.ts [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+		presenceData.state = document.querySelector(
+			".audioplayer-nowplaying__artist"
+		).textContent;
+		presenceData.smallImageKey = "live";
+		presenceData.largeImageKey =
+			document
+				.querySelector(
+					"#app > footer > div > div.audioplayer__wrapper > div.audioplayer-nowplaying.audioplayer__nowplaying > div.audioplayer-nowplaying__image > img"
+				)
+				?.getAttribute("src") ?? "logo";
```
Comment (Slowlife01) [inline websites/Q/Q-dance/presence.ts:23]:
```suggestion
			document
				.querySelector<HTMLImageElement>(
					"div.audioplayer-nowplaying__image > img"
				)
				?.src ?? "logo";
```

File: websites/Q/Q-dance/presence.ts [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
 	if (document.querySelector("svg.audioplayer-controls__icon--play")) {
-		(presenceData.details = document.querySelector(
+		presenceData.details = document.querySelector(
 			".audioplayer-nowplaying__track"
-		).textContent),
-			(presenceData.state = document.querySelector(
-				".audioplayer-nowplaying__artist"
-			).textContent),
-			(presenceData.smallImageKey = "live"),
-			(presenceData.smallImageText = (await strings).live);
```
Comment (Bas950) [inline websites/Q/Q-dance/presence.ts:21]: why text being removed?

Comment (EncryptedDev) [reply websites/Q/Q-dance/presence.ts:21]: Unresolved

Comment (EncryptedDev) [reply websites/Q/Q-dance/presence.ts:21]: Still unresolved?

Comment (DarkVillager) [reply websites/Q/Q-dance/presence.ts:21]: check line 17

**Review by ImRodry — DISMISSED** [review]

**Review by EncryptedDev — DISMISSED** [review]

**Review by ImRodry — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**